app/bridge/client.js

```diff
@@ -90,85 +121,142 @@ function createNewClient(name, version) {
     this.client = client;
     this.uuid = uuid();
 
-    this.server = null;
+    this.serverChannel = null;
```

Comment:
It is not clear that serverChannel is a better name than server. It also makes the PR a bit more confusing by merging logical changes with pure renaming.